### ISSUE TITLE
Remove added by and added date

### ISSUE THREAD

**Issue by MansMeg** [MansMeg/posteriordb#78]
This is information we get from git

**Comment by eerolinna** [comment]
Should `data_info(po)` and `model_info(po)` still return the added by and added date? If that's the case I would suggest keeping them in the info files. If not removing them is fine.

**Comment by MansMeg** [comment]
Yes. This is a little bit more long term. I will remove it from the prototype milestone.

**Comment by eerolinna** [comment]
Perhaps we should keep the current solution long term also (in that case we can just close this issue now). We won't have the git information when using a remote database. Or is there a way to easily get it from github api?

**Comment by MansMeg** [comment]
It should be relatively straight forward to get it from the github API ang from git. I would like to keep this one in the long run.

**Comment by eerolinna** [comment]
If it can be done easily I'm all for it.
